arch/ext/Zilsd.yaml

```diff
@@ -0,0 +1,23 @@
+$schema: "ext_schema.json#"
+kind: extension
+name: Zilsd
+long_name: Load/Store Pair for RV32 (Zilsd)
+description: |
+  The Zilsd extension adds RV32-only load and store pair instructions with 32-bit encoding.
+
+  **Instructions:**
+  - **ld rd, offset(rs1):** Loads a 64-bit doubleword into an aligned register pair. The lower-numbered (even) register receives bits 31:0 and the higher (odd) register receives bits 63:32.
+  - **sd rs2, offset(rs1):** Stores a 64-bit doubleword from an aligned register pair.
+
+  **Alignment and Atomicity:**
+  - If the effective address is a multiple of 8, the access is naturally aligned and will not raise a misaligned trap.
+  - If the address is a multiple of 4 but not 8, each word access is performed atomically; however, a misaligned trap may occur.
+  - For load instructions, if a trap is taken the base register retains its original value, ensuring resumable trap handling.
+  - Implementations that do not perform the doubleword load atomically must delay writeback or sequence register writes to satisfy these requirements.
+
+  This extension is independent and does not imply any other extension.
```

Comment:
This is arguably redundant with any "implies" field, or, in this case, the lack thereof. So, let's remove this line, too. :-)